systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,417 @@
+                .build();
+        resourceManager.createResource(extensionContext, true, adminClientJob.adminStrimzi().build());
+
+        Pod createPod = kubeClient().listPods("job-name", createAdminName).get(0);
+        PodUtils.waitUntilMessageIsInPodLogs(
+                createPod.getMetadata().getName(),
+                "org.apache.kafka.common.errors.ThrottlingQuotaExceededException: The throttling quota has been exceeded.",
+                Duration.ofMinutes(5).toMillis()
+        );
+        LOGGER.debug("=================\n" + kubeClient().logs(createPod.getMetadata().getName()));
+        JobUtils.deleteJobWithWait(namespaceName, createAdminName);
+
+        // Delete created topics (as they were created in random order, we have to use KafkaTopic instead of this client)
+        KafkaTopicList ktl = KafkaTopicResource.getAllKafkaTopicsWithPrefix(namespaceName, topicNamePrefix);
+        for (KafkaTopic kt : ktl.getItems()) {
+            cmdKubeClient().deleteByName(KafkaTopic.RESOURCE_SINGULAR, kt.getMetadata().getName());
+        }
+
+        // Throttling quota after performed 'alter' partitions on existing topic
+        int topicAlter = 20;
+        adminClientJob = new KafkaBasicExampleAdminClient.Builder()
+                .withAdminName(createAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(clusterName))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(topicAlter)
+                .withPartitions(1)
+                .withNamespaceName(namespaceName)
+                .withTopicOperation("create")
+                .withAdditionalConfig(getAdminClientConfig(namespaceName))
+                .build();
+        resourceManager.createResource(extensionContext, true, adminClientJob.adminStrimzi().build());
+
+        createPod = kubeClient().listPods("job-name", createAdminName).get(0);
+        PodUtils.waitUntilMessageIsInPodLogs(createPod.getMetadata().getName(), "All topics created");
+        LOGGER.debug("=================\n" + kubeClient().logs(createPod.getMetadata().getName()));
+        JobUtils.deleteJobWithWait(namespaceName, createAdminName);
+
+        // All topics altered
+        adminClientJob = new KafkaBasicExampleAdminClient.Builder()
+            .withAdminName(alterAdminName)
+            .withBootstrapAddress(KafkaResources.plainBootstrapAddress(clusterName))
+            .withTopicName(topicNamePrefix)
+            .withTopicCount(topicAlter)
+            .withPartitions(500)
+            .withNamespaceName(namespaceName)
+            .withTopicOperation("alter")
+            .withAdditionalConfig(getAdminClientConfig(namespaceName))
+            .build();
+        resourceManager.createResource(extensionContext, true, adminClientJob.adminStrimzi().build());
+        Pod alterPod = kubeClient().listPods("job-name", alterAdminName).get(0);
+        PodUtils.waitUntilMessageIsInPodLogs(
+                alterPod.getMetadata().getName(),
```

Comment:
Same as above -> get the name into some `String alterPodName` and then just add it to wait

Reply:
used only once. leaving as is.